### ISSUE TITLE
Issue with the plugin manager

### ISSUE THREAD

**Issue by brostosjoined** [bombsquad-community/plugin-manager#114]
![BombSquad_v2volc3YgU](https://user-images.githubusercontent.com/67740566/224709743-77b6d44c-e065-44ea-8ff5-3f724335742e.png)

**Comment by kingsamurai123** [comment]
Can you confirm on which occasion you received this error? Cause I don't have the issue when opening the plugin manager.
PS: I have installed all the plugins available in the plugin manager. Still don't see the error.

**Comment by Loup-Garou911XD** [comment]
Don't think this is anything to worry about, because plugin manager still continues to download when it's gui is closed
I think that's what the error is complaining about

**Comment by rikkolovescats** [comment]
Yup, I've noticed this warning too. I think it's probably cause of something in plugin manager running in asyncio after the main widget dies. It should be fun debugging this whoever feels up to it!

**Comment by rikkolovescats** [comment]
one of these should be fixed with https://github.com/bombsquad-community/plugin-manager/pull/204/files#diff-28bc375e4201ee2bf754c5e2fcdb8a9db0a36b77815a2797486ffb2433f5b181R1607

let's see if more such warnings show up
